scrapy/extensions/throttle.py

```diff
@@ -50,7 +50,7 @@ def _start_delay(self, spider):
     def _response_downloaded(self, response, request, spider):
         key, slot = self._get_slot(request, spider)
         latency = request.meta.get("download_latency")
-        if latency is None or slot is None:
+        if latency is None or slot is None or slot.throttle is False:
```

Comment:
On a first sight, it's possible to add `dont_throttle` meta key, and avoid connecting core components with extensions.

Reply:
Are we talking about implementing a downloader middleware that applies that meta key based on the `DOWNLOAD_SLOTS` setting, i.e. that setting would be managed by the core expect for that specific key? Or do you have something else in mind for the user-facing API?

Reply:
I was thinking that this feature can be unrelated to slots: just a per-request flag to disable autothrottle.
If this solves the scrapy-zyte-api issue, we can bypass all DOWNLOAD_SLOTS discussions for now :)